packages/components/pie-toggle-switch/src/toggle-switch.scss

```diff
@@ -1,3 +1,5 @@
+@use '@justeat/pie-design-tokens/dist/jet.scss' as dt;
```

Comment:
I don't think we need this. The expectation is the consumer will have the tokens already loaded into the consuming application